.travis.yml

```diff
@@ -10,6 +10,9 @@ branches:
 before_install:
 - npm config set spin false
 - npm install -g bower coveralls pr-bumper
+- bower --version
+- npm install phantomjs-prebuilt
```

Comment:
Because it's the nice thing to do. It's so much easier (3df34a3a6d4cce6d6d2bcc3ee7576d2f019c0184) to support Phantom than to make life more difficult to the grateful devs using this addon who need/want/choose to use it.